packages/ap-sports/test/unit/schedule.test.ts

```diff
@@ -0,0 +1,132 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError } from '@chainlink/ea-test-helpers'
+import type { AdapterRequest } from '@chainlink/types'
+
+import { makeExecute } from '../../src/adapter'
+import { Market } from '../../src/lib/constants'
+import { getGameCreate, getGameResolve } from '../../src/lib/schedule'
+import { Game } from '../../src/lib/types'
+
+describe('execute', () => {
+  const jobID = '1'
+  const execute = makeExecute()
+
+  process.env.API_TOKEN = 'test_api_token'
+  process.env.API_USERNAME = 'test_api_username'
+
+  describe('validating inputParameters', () => {
+    const testCases = [
+      {
+        name: "'market' is not supplied",
+        testData: {
+          data: {
+            endpoint: 'schedule',
+            sportId: 777,
+            date: 1650346477,
+          },
+        },
+      },
+      {
+        name: "'leagueId' is not supplied",
+        testData: {
+          data: {
+            endpoint: 'schedule',
+            sportId: Market.CREATE,
+            date: 1650346477,
+          },
+        },
+      },
+      {
+        name: "'date' is not supplied",
+        testData: {
+          data: {
+            endpoint: 'schedule',
+            market: Market.CREATE,
+            sportId: 777,
+          },
+        },
+      },
+      {
+        name: "'gameIds' is not an array of numbers",
+        testData: {
+          data: {
+            endpoint: 'schedule',
+            market: Market.CREATE,
+            leagueId: 777,
+            date: 1650346477,
+            gameIds: [1, 2, 'linkpool'],
+          },
+        },
+      },
+    ]
+    it.each(testCases)('errors if $name:', async ({ testData }) => {
+      try {
+        await execute(testData as AdapterRequest)
+      } catch (error) {
+        const errorResp = Requester.errored(jobID, error)
+        assertError({ expected: 500, actual: errorResp.statusCode }, errorResp, jobID)
+      }
+    })
+  })
+})
+
+describe('getGameCreate', () => {
+  it('returns the expected GameCreate', () => {
+    const event = {
+      id: '414040ab-1e79-4637-a7be-8147dcb92533',
```

Comment:
if you end up installing and using the `uuid` packagem you can replace each uuid string by `uuid()`.

Reply:
uuid() will mess up my expected IDs because they will keep changing